core/domain/question_services.py

```diff
@@ -48,6 +48,21 @@ def _create_new_question(committer_id, question, commit_message):
     create_question_summary(question.id, committer_id)
 
 
+def create_new_question_skill_link(question_id, skill_id):
+    """Creates a new QuestionSkillLink model.
+
+    Args:
+        question_id: str. ID of the question linked to the skill.
+        skill_id: str. ID of the skill to which the question is linked.
+    """
+    model = question_models.QuestionSkillLinkModel(
+        id=question_id + ':' + skill_id,
```

Comment:
I think, do, question_models.QuestionSkillLinkModel.create(question_id, skill_id)

Do we need to check that said model does not already exist yet?

Reply:
Done, also added the check.